### PR TITLE
Add APEX building to SSD L1 test

### PR DESCRIPTION
**Pull request by awolant**
Signed-off-by: Albert Wolant <awolant@nvidia.com>

### CODE REVIEW

**Comment by awolant** [comment]
Build 692171

File: qa/L1_ssd_training/test.sh [hunk 1]
```diff
@@ -10,8 +10,14 @@ test_body() {
     # workaround due to errors while pycocotools is int "pip_packages" above
     pip install -I pycocotools
 
+    #install APEX
+    git clone https://github.com/nvidia/apex
+    cd apex
```
Comment (jantonguirao) [inline qa/L1_ssd_training/test.sh]: knitpick: pushd

File: qa/L1_ssd_training/test.sh [hunk 2]
```diff
@@ -10,8 +10,14 @@ test_body() {
...
+    pip install -v --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" .
+    cd ..
```
Comment (jantonguirao) [inline qa/L1_ssd_training/test.sh]: knitpick: popd

File: qa/L1_ssd_training/test.sh (hunk 1 above)
Comment (awolant) [reply qa/L1_ssd_training/test.sh]: done

File: qa/L1_ssd_training/test.sh (hunk 2 above)
Comment (awolant) [reply qa/L1_ssd_training/test.sh]: done

**Comment by awolant** [comment]
Build 692567

**Pull request merged**